README.md

```diff
@@ -1,97 +1,37 @@
-# HTML & CSS3 Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+# Hello Microverse Project
 
-## Set-up GitHub Actions
+In this project, I set up a "Hello world" repository. No complex coding was used for this exercise. My goal here is to master all of the tools and best practices I learned about in previous steps.
 
-Please do the following **steps in this order**:
+## Getting Started
 
```

Comment:
- [ ] Kindly make sure you also complete the **## Getting Started** section as it is empty to make your **Readme** documentation more professional like adding some information about **cloning the repo locally for someone who needs its copy**, how to set up the app, install dependencies,... I suggest you take a look at [this template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) and use it as an example